rebar.config

```diff
@@ -20,7 +20,7 @@
          {git, "git://github.com/chef/epgsql-1.git", {branch, "master"}}},
 
         {pooler, ".*",
-         {git, "git://github.com/seth/pooler.git", {branch, "master"}}},
+         {git, "git://github.com/chef/pooler.git", {branch, "master"}}},
```

Comment:
do we also need to change the sha in the rebar.lock?

Reply:
It appears there is no rebar.lock in the sqerl repo.  However the sqerl reference in rebar.lock(s) for chef-server would need to be changed.  I can make an issue to do that.

Reply:
Would it be better to float the reference to sqerl in chef-server?  How often does this repo change?